js/components/portal-dashboard/student-names.tsx

```diff
@@ -12,19 +14,30 @@ export class StudentNames extends React.PureComponent<IProps> {
   render() {
     const { students, isAnonymous, isCompact } = this.props;
     const compactClass = isCompact ? css.compact : "";
+    let rowClass = css.studentNameRow;
+
     return (
       <div className={css.studentList} data-cy="student-list">
-        { students && students.map((student: any, i: number) => {
+        {students && students.map((student: any, i: number) => {
           const formattedName = isAnonymous
-                                ? student.get("name")
-                                : `${student.get("lastName")}, ${student.get("firstName")}`;
+            ? student.get("name")
+            : `${student.get("lastName")}, ${student.get("firstName")}`;
+
           return (
             <div className={`${css.studentName} ${compactClass}`} key={`student ${i}`}>
-              <div className={css.name} data-cy="student-name">{formattedName}</div>
+              <div className={rowClass} key={`student ${i}`}>
+                <div className={css.name} data-cy="student-name">{formattedName}</div>
+              </div>
             </div>
           );
-        }) }
+        })}
       </div>
     );
   }
+
+  private renderSelectionCheckbox = () => {
+    return (
+      <div className={css.checkbox}></div>
+    );
+  }
```

Comment:
Is this used?

Reply:
No, this was a left over from when I had the whole layout worked out except the student names and answers were not lining up. Removed it but may come up again later.